### DISCUSSION TITLE
What causes a window to have class "unmanaged X11"

### DISCUSSION THREAD

**Discussion by JustSimplyKyle** [hyprwm/Hyprland#1674] [General]
![image](https://user-images.githubusercontent.com/68589851/222471782-db345b2b-0990-486a-8f68-8f3d3c557f72.png)
This is weird because there is *nothing* on my second screen
Also this.
![image](https://user-images.githubusercontent.com/68589851/222471956-1e4ff88a-7518-4f08-a212-7a9401a8c2e3.png)
Why is it in monitor: -1, workspace -1

**Comment by vaxerski** [comment]
read up on the X11 protocol. It defines what an unmanaged window is. -1 means "no" because it's unmapped.

Mapped windows are those that are on your screen.
